### DISCUSSION TITLE
Some pain with start frontend

### DISCUSSION THREAD

**Discussion by nullawhale** [umputun/remark42#1185] [Q&A] [answered]
Hi, I just want to run only frontend. And following the [documentation](https://remark42.com/docs/contributing/development/) I do `npm install && npm start` inside /frontend directory.

Result in cli:
![image](https://user-images.githubusercontent.com/9190308/143450790-ec7bf6dd-8bd7-49d3-a52f-5e07f2d5e792.png)

But in browser doesn't work:
![image](https://user-images.githubusercontent.com/9190308/143450913-51a538c9-efff-487a-9254-e78929bb955f.png)

What could be the problem?

**node**: v12.18.3
**npm**: 6.14.6

**Comment by akellbl4** [comment, marked as answer]
try http://127.0.0.1:9000/web/
